### DISCUSSION TITLE
when I load tutorial file tripod.py with sofa19.06, there are errors

### DISCUSSION THREAD

**Discussion by muyi-max** [sofa-framework/sofa#2746] [Write / Run a simulation] [answered]
[ERROR]   [SofaPython] Script (file:tripod) import error
[ERROR]   [SofaPython] scene script load error.
[WARNING] [RealGUI] Failed to load D:/SOFA_v19.06.99_custom_Win64_v8.1/plugins/SoftRobots/docs/tutorials/Tripod/details/tripod.py
[main 2022-03-03T12:11:14.094Z] update#setState checking for updates
[ERROR]   [PythonScript] ImportError: No module named numpy
  File "tripod.py", line 1, in <module>
    from splib.numerics import sin, cos, to_radians
  File "D:/SOFA_v19.06.99_custom_Win64_v8.1/plugins/STLIB/python\splib\numerics\__init__.py", line 38, in <module>
    import numpy

[ERROR]   [SofaPython] Script (file:tripod) import error
[ERROR]   [SofaPython] scene script load error.
[WARNING] [RealGUI] Failed to load D:/SOFA_v19.06.99_custom_Win64_v8.1/plugins/SoftRobots/docs/tutorials/Tripod/details/tripod.py
[main 2022-03-03T12:13:37.770Z] update#setState idle

**Comment by fredroy** [comment, marked as answer]
Hello @muyi-max 
`[ERROR] [PythonScript] ImportError: No module named numpy`

you need to install numpy in your python3 environment
